src/game.py

```diff
@@ -13,21 +14,39 @@ def __init__(self, ui=UserInterface(), message=Message()):
         self.validator = Validator()
         self.symbol = SymbolOptions()
         self.ui = ui
-        self.message = message
+        self.set_language(message)
         self.game_board = self.board.starter_board
         self.player_one = "X"
         self.player_two = "O"
         self.total_marks_on_board = 0
         self.playing = True
 
+    def set_language(self, message):
+        self.message = message
+
+    def change_language(self):
+        self.ui.display_message(self.message.choose_language())
+        user_input = self.ui.get_user_input()
+        valid_user_input = self.validator.is_valid_menu_or_language_choice(user_input)
+        while valid_user_input is False:
+            self.ui.display_message(self.message.invalid_choose_language_input())
+            user_input = self.ui.get_user_input()
+            valid_user_input = self.validator.is_valid_menu_or_language_choice(
+                user_input
```

Comment:
These are the same 3 lines as 29-31. Do you think this is necessary duplication?